datadog_checks_dev/datadog_checks/dev/tooling/commands/release/testable.py

```diff
@@ -115,30 +155,22 @@ def testable(ctx, start_id, agent_version, milestone, dry_run):
         if result.code:
             abort(f'Unable to run {fetch_command}.')
 
-        if current_release_branch in result.stderr or diff_target_branch in result.stderr:
-            abort(
-                'Your repository is not sync with the remote repository. Please run git fetch in {} folder.'.format(
-                    root
-                )
-            )
+        if base_ref in result.stderr or target_ref in result.stderr:
+            abort(f'Your repository is not sync with the remote repository. Please run git fetch in {root!r} folder.')
 
-        # compare with the local tag first
-        reftag = f"{'refs/tags/'}{current_release_branch}"
-        result = run_command(diff_command.format(reftag, diff_target_branch), capture=True)
+        reftag = f"refs/tags/{base_ref}"
+        result = run_command(diff_command.format(reftag, target_ref), capture=True)
         if result.code:
-            # if it didn't work, compare with a branch.
```

Comment:
nit- could you add back a comment here saying what is happening in this if statement?

Reply:
The latest commit changes the error handling logic, let me know if it's clearer know. :-)

Reply:
👍